app/src/main/res/layout/activity_enter_mnemonic.xml

```diff
@@ -12,35 +12,21 @@
         android:layout_height="wrap_content"
         android:background="@color/colorPrimary">
 
-        <com.google.android.material.textview.MaterialTextView
-            android:id="@+id/welcomeTextView"
-            android:layout_width="wrap_content"
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/createWalletFromMnemonicButton"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginStart="16dp"
-            android:layout_marginTop="24dp"
+            android:layout_marginTop="8dp"
```

Comment:
how does this constant play with various size classes?

Reply:
nothing to worry, was making tweaks to left over code! This screen is an old design and will be removed eventually.